{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -303,17 +308,15 @@
 
     aws_s3_domain = AWS_S3_CUSTOM_DOMAIN or f"{AWS_STORAGE_BUCKET_NAME}.s3.amazonaws.com"
     # Default file storage is private
+
     PRIVATE_MEDIAFILES_LOCATION = AWS_LOCATION + "/media"
-    DEFAULT_FILE_STORAGE = "{{ cookiecutter.project_slug }}.utils.storages.PrivateMediaStorage"
-    STATICFILES_STORAGE = "{{ cookiecutter.project_slug }}.utils.storages.StaticRootS3Boto3Storage"
+    STATIC_FILES_LOCATION = AWS_LOCATION + "/static"
+    DEFAULT_FILE_STORAGE = "miles_for_military.core.handlers.PrivateMediaStorage"
+#   #STATICFILES_STORAGE = "miles_for_military.core.handlers.StaticStorage"
```

Comment:
Need to replace `miles_for_military` here with `{{ cookiecutter.project_slug }}`.

As mentioned above, we've never used S3 for staticfiles, so we can probably remove `STATICFILES_STORAGE` here, unless there's some reason we want to make S3 Static files work.